drivers/backup/portworx/portworx.go

```diff
@@ -927,6 +927,50 @@ func (p *portworx) GetVolumeBackupIDs(
 	return volumeBackupIDs, nil
 }
 
+// ListBackupCRs lists all the Backup CRs present under given namespace
+func (p *portworx) ListBackupCRs(
+	ctx context.Context,
```

Comment:
ctx?

Reply:
This will list all the backup CRs that got created under a namespace, the helper methods calling this method could be used to switch the context to the cluster where the CRs are getting created

Reply:
My bad. Why ctx context.Context?

Reply:
It's refactored and removed